stylesheets/blue/components/_nav.scss

```diff
@@ -176,6 +179,17 @@ $Nav-item-color-hover: $Theme-color-yellowOrange;
   }
 }
 
+.Nav-itemNoUnderline {
```

Comment:
I would suggest `itemWithoutUnderline`, but I'm ok if you want to keep it this way.